extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/WorkerContext.java

```diff
@@ -73,4 +74,6 @@
   DruidNode selfNode();
 
   Bouncer processorBouncer();
+
+  Emitter emitter();
```

Comment:
Can the emitter be nullable? If so, appropriate checks and annotations would be helpful

Reply:
We are getting this from `TaskToolBox` and the emitter there isn't nullable